ci/setup_env.sh

```diff
@@ -121,7 +121,7 @@ conda list pandas
 # Make sure any error below is reported as such
 
 echo "[Build extensions]"
-python setup.py build_ext -q -i
+python setup.py build_ext -q -i -j2
```

Comment:
Any reason for using 2 processes? I'd set `-j0` and use as many processes are cores available.

Reply:
Is this documented somewhere? I tried locally and -j0 doesn't seem to parallelize anything

Reply:
You're right, not sure why I was sure this could be automatically parallelized to the number of cores. I think it was intended to have this implemented, see https://github.com/python/cpython/blob/master/Lib/distutils/command/build_ext.py#L453 but in practice there is no way to make `parallel` equal to `True`.

Given that, I'd use:
```
-j `nproc`
```

Reply:
does the parallel extensions flag not work? is -j the new thing?

Reply:
Parallel and j are the same thing for distutils:

https://github.com/python/cpython/blob/03c8e5d46673e4704c8dc955d52735966ffdc2a4/Lib/distutils/command/build_ext.py#L87

And have the same requirements for specifying an integer (the assignment to `self.parallel` as shown in Marc's link I think can be misleading). 

Here is the GNU make documentation on that:

https://www.gnu.org/software/make/manual/html_node/Parallel.html#Parallel

Reply:
@datapythonista not sure that works for macOS unless you know of an alternate syntax?

Reply:
I don't know much about mac, but `-j $(nproc)` is equivalent, and works when the shell is not bash I think

Reply:
Hmm so I don't think nproc is available by default in macOS, so reverting to 2 for now (though definitely a nice idea)

Reply:
it looks like the OSX equivalent is `sysctl -n hw.ncpu`, but that's one of the namespaces that apple is liable to change every release